### ISSUE TITLE
Lack of newlines between some of the stanzas causes some obo parsers to fail

### ISSUE THREAD

**Issue by krassowski** [DiseaseOntology/HumanDiseaseOntology#481]
This works fine:
```
[Term]
id: DOID:4286
name: sebaceous basal cell carcinoma
synonym: "skin basosebaceous basal cell carcinoma" EXACT []
is_a: DOID:2513 ! basal cell carcinoma

[Term]
id: DOID:4287
name: external ear basal cell carcinoma
synonym: "basal cell carcinoma of external ear" RELATED []
synonym: "Basal cell carcinoma of the External ear" EXACT [NCI2004_11_17:C6082]
xref: NCI:C6082
xref: UMLS_CUI:C1333491
is_a: DOID:4288 ! external ear carcinoma
```

But this causes an error:

```
[Term]
id: DOID:4288
name: external ear carcinoma
def: "An external ear cancer that has_material_basis_in abnormally proliferating cells derives_from epithelial cells." [url:http\://en.wikipedia.org/wiki/Carcinoma]
synonym: "carcinoma of external ear" RELATED []
synonym: "carcinoma of the External ear" EXACT [NCI2004_11_17:C6081]
xref: NCI:C6081
xref: UMLS_CUI:C1333492
is_a: DOID:5665 ! external ear cancer
[Term]
id: DOID:4289
name: micronodular basal cell carcinoma
synonym: "skin micronodular basal cell carcinoma" EXACT []
is_a: DOID:2513 ! basal cell carcinoma
```

**Comment by krassowski** [comment]
Ok, I see that it was fixed in https://github.com/DiseaseOntology/HumanDiseaseOntology/commit/52c123df04909c99c438a50352b19abfcbc8f12a, but only for `doid-non-classified.obo`, not for  	`HumanDO.obo`.

I am confused, what are the differences between the files? I read that:

> In the current DO production environment, the HumanDO.obo and HumanDO.owl files are equivalent to the
> doid-non-classified.obo and doid-non-classified.owl files produced in this GitHub repository.

But the files in the repository differ; why is that?

**Comment by lschriml** [comment]
Hello, Thank you for pointing this out. 
You are correct the doid-non-classified.obo and HumanDO.obo should match.
The only difference is the name. 
We originally named our files as 'HumanDO'... , when we moved to producing the OWL files,
this process involved a new set of files, where the file names had to conform to the obo foundry 
naming schema. In order to still provide our original (and published) file names, we maintain the two sets of files. 

Apologies for the disconnect, we had a parsing error in the last build with the doid-non-classified.obo file, this should have been copied to the HumanDO.obo file. 
I have now updated the file. 

Cheers,
Lynn
